content/auth/token.textile

```diff
@@ -275,6 +329,15 @@ ably.auth.requestToken({ clientId: 'client@example.com' }, (err, token) => {
 });
 ```
 
+```[python]
+rest = AblyRest(key='{{API_KEY}}')
+token_request_data = {
```

Comment:
the word 'data' kinda implies request body, rather than request params
```suggestion
token_request_params = {
```

Reply:
https://github.com/ably/docs/pull/2129/commits/a8e480abdc556b03e67257938fa30e8fe540f336